premerge/pdb.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: policy/v1
```

Comment:
I would prefer a more descriptive file name than "pdb.yaml".

Reply:
Updated to `pod-disruption-budget.yaml`.